src/main/java/frc/trigon/robot/commands/factories/GeneralCommands.java

```diff
@@ -24,6 +24,12 @@
 public class GeneralCommands {
     public static boolean IS_BRAKING = true;
 
+    public static Command getVisualizeNoteShootingCommand() {
+        return new InstantCommand(
+                () -> new VisualizeNoteShootingCommand()
+                        .schedule()).onlyIf(() -> RobotHardwareStats.isReplay() || MotorSubsystem.isExtensiveLoggingEnabled());
```

Comment:
I believe in my PR I change extensive logging to alos enable if it is a replay. Double check and if so, remove that part